__tests__/sum.test.js

```diff
@@ -0,0 +1,68 @@
+const Q = require('q');
```

Comment:
With regards to the name of this file, I'm a bit curious about why you named it with `sum.test.js`. Is it some convention defined in Jest framework?